deployment/gtfs_delta.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Name: gtfs_delta; Type: TABLE; Schema: public; Owner: transit_indicators; Tablespace:
+--
+
+CREATE TABLE public.gtfs_delta (
+  id text PRIMARY KEY,
+  deltaType integer NOT NULL,
+  geom geometry(LineString,4326) NOT NULL,
+);
+
+ALTER TABLE public.station_csv OWNER TO transit_indicators;
```

Comment:
This should probably be broken out into a separate commit, it looks like it's a bug fix.